frontend/src/app/(routes)/transfers/SendPage.tsx

```diff
@@ -0,0 +1,321 @@
+                      <InputAdornment position="start">
+                        {selectedAsset?.displayDenom}
+                      </InputAdornment>
+                    ),
+                    sx: {
+                      input: {
+                        color: 'white',
+                        fontSize: '14px',
+                        padding: 2,
+                      },
+                    },
+                  }}
+                  error={!!errors.amount}
+                />
+              )}
+            />
+
+            <div
+              className={`text-red-500 text-xs px-4 opacity-95 mt-1 min-h-[24px]`}
+            >
+              {!sortedAssets.length
+                ? 'No Assets Found.'
+                : !selectedAsset
+                  ? 'Please select an Asset'
+                  : errors.amount
+                    ? errors.amount?.message
+                    : ''}
+            </div>
+            <div className="min-h-[96px]">
+              <div className="flex items-center gap-2  mb-2">
+                <div className="flex items-center text-sm not-italic font-normal leading-[normal]">
+                  Memo
+                </div>
+                <Image
+                  onClick={() => setOpenMemo((openMemo) => !openMemo)}
+                  src="/drop-down-icon.svg"
+                  className='cursor-pointer'
+                  width={16}
+                  height={16}
+                  alt=""
+                />
+              </div>
+              {openMemo ? (
+                <Controller
+                  name="memo"
+                  control={control}
+                  render={({ field }) => (
+                    <TextField
+                      className="bg-[#FFFFFF0D] rounded-2xl"
+                      {...field}
+                      required
+                      fullWidth
+                      size="small"
+                      placeholder="Enter Memo here"
+                      sx={{
+                        '& .MuiTypography-body1': {
+                          color: 'white',
+                          fontSize: '12px',
+                          fontWeight: 200,
+                          height: '100px',
+                        },
+                        '& .MuiOutlinedInput-notchedOutline': {
+                          border: 'none',
+                        },
+                      }}
+                      InputProps={{
+                        sx: {
+                          input: {
+                            color: 'white',
+                            fontSize: '14px',
+                            padding: 2,
+                          },
+                        },
+                      }}
+                      error={!!errors.memo}
+                    />
+                  )}
+                />
+              ) : null}
+            </div>
+          </div>
+        </div>
+        <div className="h-full flex gap-10 items-center">
+          <button
```

Comment:
button also, because most of the places we use the same functionality  loading and text and type submit always